Sources/MongoSwift/Operations/DropIndexesOperation.swift

```diff
@@ -15,6 +15,12 @@ public struct DropIndexOptions: Encodable {
     }
 }
 
+/// The result of performing a "dropIndexes" command.
+public struct DropIndexesResult: Decodable {
+    /// The number of indexes that were present in the collection prior to performing the drop.
+    public let nIndexesWas: Int
```

Comment:
let me know if you have naming opinions on this... I considered `previousIndexCount` or `indexCountBeforeDrop` or something but those felt kinda verbose

Reply:
I think it's okay to use the name as it is in the actual response.